registration-processor/core-processor/registration-processor-uin-generator-stage/src/test/java/io/mosip/registration/processor/stages/uigenerator/UinGeneratorStageTest.java

```diff
@@ -429,6 +432,67 @@ public void testUinGenerationResponseNull() throws Exception {
 		MessageDTO result = uinGeneratorStage.process(messageDTO);
 		//assertFalse(result.getInternalError());
 	}
+
+	@Test
```

Comment:
Is this test case for retry count is less than max count?
any other test case for which exceed max count?